.github/workflows/pr_approval.yml

```diff
@@ -1,25 +1,34 @@
+# This workflow checks that the PR has been approved by 2+  members of the committee listed in `pull_requests.toml`.
+#
+# Run this pull request when a request review is submitted / dismissed.
+# Note that an approval can be dismissed, and this can affect the job status.
+# We currently trust that contributors won't make significant changes to their PRs after approval, so we accept
+# changes after approval.
+#
+# We still need to run this in the case of a merge group, since it is a required step. In that case, the workflow
+# is a NOP.
 name: Check PR Approvals
-
-# For now, the workflow gets triggered only when a review is submitted
-# This technically means, a PR with zero approvals can be merged by the rules of this workflow alone
-# To protect against that scenario, we can turn on number of approvals required to 2 in the github settings
-# of the repository
 on:
+  merge_group:
+  pull_request:
```

Comment:
Why are we making this change as part of the merge-group enablement?

Reply:
1. There's no safety barrier today.
2. I will revert this though because of the duplicated job. 

```suggestion
```